src/utils/p-try-each.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const pTryEeach = async (iterable) => {
```

Comment:
Can you create a module for this?

Reply:
Yes, I can and I will. Can we merge this as it is and then update it later to use the module that I'll create?

Reply:
you have a typo in the fn name `pTryEeach`should be `pTryEach`

Reply:
Oops 🤦‍♂️